Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -248,7 +248,7 @@ def get_activities_request(self, created_after: str = None, user_emails: str = N
     def get_threats_request(self, content_hash=None, mitigation_status=None, created_before=None, created_after=None,
                             created_until=None, created_from=None, updated_from=None, resolved='false', display_name=None,
                             query=None, threat_ids=None, limit=20, classifications=None, site_ids=None, rank=None,
-                            include_resolved_param=True):
+                            include_resolved_param=True, incidentStatuses=None):
```

Comment:
```suggestion
                            include_resolved_param=True, incident_statuses=None):
```
why are we changing the convention?